### PR TITLE
Enable sle15sp6 virt MU

### PR DESCRIPTION
**Pull request by tbaev**
The VR runs use RC1 and have some issues, I am looking into them but I think it is best to resolved them in new tickets/PRs.

- Related ticket: https://progress.opensuse.org/issues/153916
- Needles: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/-/merge_requests/1661
- Verification run: 

[12-SP5-KVM](http://openqa.qam.suse.cz/tests/70819#dependencies)
[12-SP5-XEN](http://openqa.qam.suse.cz/tests/70902#dependencies)

[15-SP6-KVM](http://openqa.qam.suse.cz/tests/70747#dependencies) issue: [poo#157507](https://progress.opensuse.org/issues/157507)
[15-SP6-XEN](http://openqa.qam.suse.cz/tests/70627#dependencies)

[HyperV](https://openqa.suse.de/tests/13959830)
[VMware](https://openqa.suse.de/tests/13959831)

### CODE REVIEW

File: lib/virt_autotest/common.pm [hunk 1]
```diff
@@ -217,6 +240,15 @@ if (get_var("REGRESSION", '') =~ /xen/) {
             distro => 'SLE_15',
             location => 'http://mirror.suse.cz/install/SLP/SLE-15-SP5-Full-LATEST/x86_64/DVD1/',
             linuxrc => 'ifcfg="eth0=192.168.122.109/24,192.168.122.1,192.168.122.1"'
+        },
+        sles15sp6 => {
+            name => 'sles15sp6',
+            extra_params => '--os-variant sle15-unknown',
+            macaddress => '52:54:00:78:73:a8',
```
Comment (RoyCai7) [inline lib/virt_autotest/common.pm:247]: 52:54:00:78:73:a8 is duplicated with sles15sp5HVM

File: lib/virt_autotest/common.pm [hunk 2]
```diff
@@ -217,6 +240,15 @@ if (get_var("REGRESSION", '') =~ /xen/) {
...
+            ip => '192.168.122.110',
```
Comment (RoyCai7) [inline lib/virt_autotest/common.pm:248]: 192.168.122.110 is the same as sles15sp4PV

File: data/virtualization/autoyast/guest_15.xml.ep [hunk 3]
```diff
@@ -364,16 +364,15 @@
...
-logger "IP is written into file"
-curl -k -u root:{{PASS}} -T /root/$(hostname) sftp://{{SUT_IP}}/tmp/guests_ip/ --ftp-create-dirs
-exit 0
-EOF
-chmod u+x /etc/init.d/after.local
 ]]></source>
 </script>
-</post-scripts>
+<script>
+<filename>PU0-cv-users.sh</filename>
```
Comment (alice-suse) [inline data/virtualization/autoyast/guest_15.xml.ep:370]: Just curious. What does this file name stand for ?

File: lib/virt_autotest/common.pm [hunk 4]
```diff
@@ -226,7 +258,7 @@ if (get_var("REGRESSION", '') =~ /xen/) {
             delete $guests{$guest} unless grep { $_ eq $guest } @allowed_guests;
         }
     } elsif (is_sle('=12-SP5')) {
-        my @allowed_guests = qw(sles12sp5 sles15sp1 sles15sp5);
+        my @allowed_guests = qw(sles12sp5 sles15sp1 sles15sp5 sles15sp6);
```
Comment (RoyCai7) [inline lib/virt_autotest/common.pm:261]: Please help delete sles15sp1, last time, I forgot to delete it, thanks.

**Comment by alice-suse** [comment]
> The VR runs use RC1 and have some issues, I am looking into them but I think it is best to resolved them in new tickets/PRs.

Sure, of course :). If you open new tickets, suggest to add them as sub-task to current ticket, so that we can have an umbrella ticket to track them all.

LGTM, only one minor question out of curiosity.

File: data/virtualization/autoyast/guest_15.xml.ep (hunk 3 above)
Comment (tbaev) [reply data/virtualization/autoyast/guest_15.xml.ep:370]: I based it on the name of the original script <filename>PU1-cv-users.sh</filename> , not sure what the abbreviation stands for it was introduced in this [PR](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/14921)

**Review by RoyCai7 — CHANGES_REQUESTED** [review]

**Comment by RoyCai7** [comment]
LGTM

**Pull request merged**